content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -29,6 +29,21 @@ Kubernetes, on Linux:
 - [Docker Engine](#docker)
 - [Mirantis Container Runtime](#mcr)
 
+{{< note >}}
+Docker container runtime (Dockershim) has been [deprecated](
```

Comment:
```suggestion
Docker container runtime (Dockershim) has been [deprecated](
```
Implies that dockershim and Docker container runtime are the same, please revise